### ISSUE TITLE
Ruffle Error on the official test page of adobe

### ISSUE THREAD

**Issue by CreaZyp154** [ruffle-rs/ruffle#2042]
Here's the raw error message

# Error Info
Error name: TypeError
Error message: bi[e] is not a function
Error stack:
```
TypeError: bi[e] is not a function
    at Function.hi (<anonymous>:3:32170)
    at https://wwwimages2.adobe.com/ubi/template/identity/lib/prototype.js:209:23
```

# Player Info
Allows script access: false

# Page Info
Page URL: https://get.adobe.com/fr/flashplayer/about/

# Browser Info
Useragent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36 Edg/87.0.664.66
OS: Win32

# Ruffle Info
Version: 0.1.0
Name: nightly 2020-12-19
Channel: nightly
Built: 2020-12-19T00:34:27.448Z
Commit: 042e9a444b98f7dbf920a070ffe240ad01e7cbe4

**Comment by parkerlreed** [comment]
This was JSUT reported and closed in favor of a couple past issues. https://github.com/ruffle-rs/ruffle/issues/2038

**Comment by Toad06** [comment]
Fixed by #2051 (the file itself will not load due to CORS issues).
